docs/runtime-upgrade.md

```diff
@@ -43,15 +43,15 @@ This must be done before releasing a new version of any of our runtimes to ensur
 
     - Click on **propose**
 
-1. Voting phase. You should make 3 votes in favor with 3 diferent people (modify the CFG amount before to a low number).
+1. Voting phase. You should make 3 votes in favor of 3 different people (modify the CFG amount before to a low number).
```

Comment:
That correction is not actually correct, since the desired semantic here is that 3 different accounts vote `Aey`, note that someone "should make 3 votes in favour of 3 different people".

```suggestion
1. Voting phase. Three different account should vote in favour of the proposal (modify the CFG amount before to a low number).
```

Reply:
I'm also not sure we need 3 separate 👍 , 1 should be enough? 🤔  cc @wischli

Reply:
> I'm also not sure we need 3 separate 👍 , 1 should be enough? 🤔 cc @wischli

Our local chain specs setup 4 genesis council members

https://github.com/centrifuge/centrifuge-chain/blob/0387385954349e3ca8b2c4d84e4e17b16bbbc5db/src/chain_spec.rs#L541-L543

Since both proposing an external referendum as well as fast-tracking requires at least half of the council, two votes would suffice. I suppose this is a remnant of previously requiring at least 75% of the council.

On that note, our runtime upgrade scripts votes with three members:

https://github.com/centrifuge/centrifuge-chain/blob/0387385954349e3ca8b2c4d84e4e17b16bbbc5db/scripts/js/upgrade/index.js#L87-L93

Reply:
The number of votes must match the threshold set when proposing. The document currently states 3 (75%) which is outdated
```diff
1. In Governance -> Council -> Motions, click on **propose motion**.
-    - At **umbral**, put `3` (a 75% of the total which is 4).
+    - At **umbral**, put `2` (a 50% of the total which is 4).
```